README.md

```diff
@@ -57,9 +61,46 @@ uv pip install ai-vocabulary-builder
 
 After installation, run `aivoc notebook` to open the application in your browser.
 
-## Docker
+### Option 2: Docker Installation
 
-For easier setup and a consistent environment, you can also use Docker. See the [CONTRIBUTING.md](CONTRIBUTING.md) file for instructions on building and running the Docker image.
+If you prefer using Docker, we provide a containerized version of AI Voc Builder:
+
+1. Clone the repository
+```bash
+git clone https://github.com/piglei/ai-vocabulary-builder.git
+cd ai-vocabulary-builder
+```
+
+2. Start the application
+
+```bash
+make build  # Build the Docker image
```

Comment:
It is not common practice to recommend users to use the Makefile to run the application, the Makefile is more like a tool for developers only. So it would be better to move some of these instructions to `CONTRIBUTING.md`.